documentation/examples/prometheus-service.yml

```diff
@@ -54,6 +54,13 @@ items:
           volumeMounts:
           - name: config-volume
             mountPath: /etc/prometheus
+          resources:
+            requests:
+              cpu: 20m
+              memory: 50Mi
+            limits:
+              cpu: 40m
+              memory: 100Mi
```

Comment:
What effect is this going to have? Based on the title, it'll bound usage, but these seem very low. Have you done some analysis of what load it can support?

Reply:
My analysis is as trivial as it could have possibly been. I monitored a prometheus pod in a cluster running over the course of the day with only itself as a target, multiplied the resource utilization by 2 for requests, and by 2 again for limits. While this analysis isn't thorough, I'm happy to follow up in the future with values based on predictable load. A user can always change these values manually should they begin to see performance issues, but I agree that we shouldn't set this value so low that 50% of users would encounter this issue.